### ISSUE TITLE
How would I add testing data?

### ISSUE THREAD

**Issue by qwertpi** [jg-fisher/diabetesNeuralNetwork#1]
How would I use some of the data for testing instead of training so I can check for overfitting?

**Comment by jg-fisher** [comment]
The sklearn package has a function called train_test_split that will do this.  I recently updated the repo to include a validation set.
